Formula/riak-ts.rb

```diff
@@ -0,0 +1,37 @@
+class RiakTs < Formula
+  desc "NoSQL time series database optimized for IoT and Time Series data."
+  homepage "http://basho.com/products/riak-ts/"
+  url "http://s3.amazonaws.com/downloads.basho.com/riak_ts/1.3/1.3.0/osx/10.8/riak-ts-1.3.0-OSX-x86_64.tar.gz"
```

Comment:
You're downloading a binary distribution, which is not allowed. Need to install from source.